### PR TITLE
Fix assessment views when its end time is empty

### PR DESCRIPTION
**Pull request by minhtule**

### CODE REVIEW

File: app/views/course/assessment/assessments/_assessment.html.slim [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
   td
   td = format_datetime(assessment.start_at)
-  td = format_datetime(assessment.end_at)
+  td = format_datetime(assessment.end_at) unless assessment.end_at.blank?
```
Comment (lowjoel) [inline app/views/course/assessment/assessments/_assessment.html.slim]: `if assessment.end_at.present?`

**Comment by lowjoel** [comment]
Actually maybe we should just omit the end-at line if it isn't defined.

**Comment by minhtule** [comment]
I think it's better to still show it and put a dash there ` - `

**Comment by lowjoel** [comment]
But your PR does not do this?

**Comment by minhtule** [comment]
So what do we agree on? Empty, dash or removed completely?

**Comment by lowjoel** [comment]
Let's leave it empty first.

**Comment by minhtule** [comment]
Thanks!

**Pull request merged**